server/utils/buffer.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  indexOf: function(buffer, charCode) {
```

Comment:
astoundingly, buffers have a `length` property and can be indexed into, but lack an `indexOf` method: http://nodejs.org/api/buffer.html